### PR TITLE
Use Azure Artifacts feeds instead of MyGet

### PR DESCRIPTION
**Pull request by garath**
As part of [moving to Azure Artifacts from MyGet](https://github.com/dotnet/arcade/blob/master/Documentation/MigrationPlan/MyGetFeeds.md), point all MyGet feed sources to new Azure Artifacts feed.

### CODE REVIEW

**Comment by markwilkie** [comment]
@safern - is this something you could help with?

File: dir.props [hunk 1]
```diff
@@ -78,8 +78,8 @@
   <PropertyGroup>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
-      https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
-      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json;
+      https://dotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json;
```
Comment (safern) [inline dir.props:82]: why do we need the blob? I would expect both the `myget-legacy` feed and the blob to contain the same assets, right?

File: src/NuGet.config [hunk 2]
```diff
@@ -4,8 +4,8 @@
   <!-- The command-line doesn't need it, but the IDE does.                    -->
   <packageSources>
     <clear/>
-    <add key="myget.org dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
-    <add key="myget.org dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="myget.org dotnet-buildtools" value="pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
+    <add key="myget.org dotnet-core" value="https://dotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json" />
```
Comment (safern) [inline src/NuGet.config:8]: Same question here.

**Comment by markwilkie** [comment]
cc/ @riarenas @mmitche

**Comment by ghost** [comment]
I'm not familiar with either of those feeds as they were created just for this effort. I can try and hack something up to determine diffs between both feeds if @garath doesn't know the answer.

**Comment by markwilkie** [comment]
Thanks @riarenas, no worries.  

@garath - would you mind answering @safern?

**Comment by ghost** [comment]
and here I thought I had my one man hackathon project.

File: dir.props (hunk 1 above)
Comment (garath) [reply dir.props:82]: > why do we need the blob? I would expect both the myget-legacy feed and the blob to contain the same assets, right?

The dotnet-core feed is a very large feed, so I opted to move it to sleet instead for ease of setup. The only packages in the dotnetmygetlegacy feed are those from dotnet-core. 

I would be open to moving some of those packages into the myget-legacy feed and eliminating this line in corefx, if you could identify the packages and version.

**Comment by safern** [comment]
I don't mind merging as is. I was just curious why we had 2 feeds instead of one 😄

**Review by safern — APPROVED** [review]

**Comment by markwilkie** [comment]
Thanks @safern !  Looks like you're cleared to hit the merge button @garath

**Comment by garath** [comment]
> Looks like you're cleared to hit the merge button

Unfortunately I do not have merge rights. @safern, do you mind clicking that?

**Pull request merged**